.github/workflows/build-and-publish.yml

```diff
@@ -0,0 +1,92 @@
+name: Docker Image CI
+
+on:
+  push:
+    tags:
+     - v**
+  pull_request:
+    branches: [ master ]
```

Comment:
If I understand this right, we are triggering the build process on (I assume) merged pull requests? Or on all pull requests targeting master?

Reply:
any pull request targeting master -- this is a PR verification step mostly. Any actual publishing steps are skipped for these builds.

Reply:
Sounds excellent - it would also allow us to include some testing steps at a later point, right?